.gitignore

```diff
@@ -2,3 +2,4 @@
 /_cicd-intermediates
 /.build
 .DS_Store
+.idea/
```

Comment:
Just a nit here. Add a new line at the end of file, commit it and push so we stay consistent with the CRLF line endings.